Formula/dart-beta.rb

```diff
@@ -2,10 +2,9 @@
 # frozen_string_literal: true
 
 class DartBeta < Formula
-  desc "Dart Beta SDK"
```

Comment:
This seems...weird.

Is it filling in automagically?

Reply:
I agree, but brew audit has a lint that `desc` doesn't start with the class name. so `class Dart < formula\n desc "Dart SDK"` is a lint violation:

```
Library/Taps/dart-lang/homebrew-dart/dart.rb:5:9: C: [Correctable] Description shouldn't start with the formula name.
  desc "Dart SDK"
        ^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

`brew info` sort of adds that (it kind of does, but on the line above):

```
dart: stable 2.13.1, HEAD
SDK
```

So I changed it for `DartBeta` to be consistent with `Dart`, although personally I disagree that the lint improves the description.